src/platform/utilities/feature-toggles/featureFlagNames.json

```diff
@@ -282,7 +282,6 @@
   "vaDependentsV2Banner": "va_dependents_v2_banner",
   "vaOnlineScheduling": "va_online_scheduling",
   "vaOnlineSchedulingBookingExclusion": "va_online_scheduling_booking_exclusion",
-  "vaOnlineSchedulingBreadcrumbUrlUpdate": "va_online_scheduling_breadcrumb_url_update",
```

Comment:
Wouldn't this break usage of `vaOnlineSchedulingBreadcrumbUrlUpdate` by other teams? 

e.g. https://github.com/department-of-veterans-affairs/vets-website/blob/a310ea3453e1cb30c43d29f8ac096faf4cea0c38/src/applications/check-in/components/LinkList.jsx#L44 and https://github.com/department-of-veterans-affairs/vets-website/blob/a310ea3453e1cb30c43d29f8ac096faf4cea0c38/src/applications/check-in/pre-check-in/pages/Error/index.jsx#L23

Reply:
We also need to update static pages it looks like: https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/applications/static-pages/cta-widget/components/messages/VAOnlineScheduling.jsx#L22-L24. They actually use this to choose the correct URL a few lines down